inginious/frontend/static/js/checked-list-group.js

```diff
@@ -12,6 +12,8 @@ $(function()
             $checkbox.attr('name', $widget.data('name'));
         if($widget.data('value'))
             $checkbox.attr('value', $widget.data('value'));
+        if($widget.data('class'))
+            $checkbox.addClass($widget.data('class'));
```

Comment:
Is there no other way than modifying this file ? We may forget to make this change in case we update boostrap as it was fetch as is from the Internet.

Reply:
I will just revert back to the way it was selected before